docs/userGuide/syntax/includes.md

```diff
@@ -56,6 +56,35 @@ When setting the `id` of a fragment, be careful not to clash with heading anchor
 </box>
 
 <include src="panels.md#script_and_styles_warning"></include>
+<div id="baseUrl-warning">
+<box type="warning" header="Add `{{ '{{ baseUrl }}' }}` to make your URLs absolute links if they may be reused in different contexts">
+
+Make an internal relative link an absolute link by adding `{{ '{{ baseUrl }}' }}` in front of the path. This allows the link to always point to the same target. Keep this in mind when putting content with links that is reused (eg: via `<include>`) as when your content is re-used a relative link may no longer point to where you want it to.
+
+<div id="baseUrl-example">
+<panel header="Example of using absolute links in `<include>`" type="info">
+
+The file `folder1/file1.md` contains a link to `folder1/target.html`. The file `folder2/file2.md` contains a fragment from `folder1/file1.md` using `<include>`.
+
+**In `folder2/file2.md`:** 
+```
+<include src="folder1/file1.md" />
+```
+
+<box type="success" header="Positive example">
+
+To ensure that the link will still point to `folder1/target.html`, use an **absolute link** in `folder1/file1.md` as such: `{{ '{{baseUrl}}' }}/folder1/target.html`.
+</box>
+
+<box type="wrong" header="Negative example">
+
+If a relative link `target.html` is used, the link will point to `folder2/target.html` instead of `folder1/target.html`
```

Comment:
```suggestion
If a relative link `target.html` is used, the link in `folder2/file2.md` will point to `folder2/target.html` instead of `folder1/target.html`
```